x-pack/plugin/esql/qa/testFixtures/src/main/resources/unmapped-load.csv-spec

```diff
@@ -0,0 +1,636 @@
+2024-10-23T13:51:54.732Z | Connection error?
+2024-10-23T13:33:34.937Z | 42
+2024-10-23T12:27:28.948Z | Connected to 10.1.0.2!
+2024-10-23T12:15:03.360Z | Connected to 10.1.0.3!
+;
+
+unmappedFieldDoesNotAppearLast
+required_capability: unmapped_fields
+required_capability: optional_fields
+
+SET unmapped_fields="load"\;
+FROM partial_mapping_sample_data
+| SORT @timestamp DESC
+| LIMIT 1
+;
+
+@timestamp:date         |client_ip:ip   |event_duration:long |message:keyword
+2024-10-23T13:55:01.543Z|173.21.3.15    |1756466             |Connected to 10.1.0.1!
+;
+
+fieldDoesNotExistSingleIndex
+required_capability: unmapped_fields
+required_capability: optional_fields
+
+SET unmapped_fields="load"\;
+FROM partial_mapping_sample_data
+| KEEP @timestamp, foo
+| SORT @timestamp DESC
+;
+
+@timestamp:date           |  foo:keyword
+2024-10-23T13:55:01.543Z  |  null
+2024-10-23T13:53:55.832Z  |  null
+2024-10-23T13:52:55.015Z  |  null
+2024-10-23T13:51:54.732Z  |  null
+2024-10-23T13:33:34.937Z  |  null
+2024-10-23T12:27:28.948Z  |  null
+2024-10-23T12:15:03.360Z  |  null
+;
+
+fieldIsUnmappedSingleIndex
+required_capability: unmapped_fields
+required_capability: optional_fields
+
+SET unmapped_fields="load"\;
+FROM partial_mapping_sample_data
+| KEEP @timestamp, message, unmapped_message
+| SORT @timestamp DESC
+;
+
+@timestamp:date          | message:keyword        | unmapped_message:keyword
+2024-10-23T13:55:01.543Z | Connected to 10.1.0.1! | Disconnected from 10.1.0.1
+2024-10-23T13:53:55.832Z | Connection error?      | Disconnection error
+2024-10-23T13:52:55.015Z | Connection error?      | Disconnection error
+2024-10-23T13:51:54.732Z | Connection error?      | Disconnection error
+2024-10-23T13:33:34.937Z | 42                     | 43
+2024-10-23T12:27:28.948Z | Connected to 10.1.0.2! | Disconnected from 10.1.0.2
+2024-10-23T12:15:03.360Z | Connected to 10.1.0.3! | Disconnected from 10.1.0.3
+;
+
+# Kept this test disabled to show the difference from IGNORE_🐔: if everywhere unmapped and not mentioned, a field (message) won't show
+# up at all.
+fieldIsUnmappedButSourceIsDisabledSingleIndex-Ignore
+required_capability: source_field_mapping
+required_capability: unmapped_fields
+required_capability: optional_fields
+
+SET unmapped_fields="load"\;
+FROM partial_mapping_no_source_sample_data
+;
+
+@timestamp:date          | message:keyword
+2024-10-23T13:55:01.543Z | null
+2024-10-23T13:53:55.832Z | null
+2024-10-23T13:52:55.015Z | null
+2024-10-23T13:51:54.732Z | null
+2024-10-23T13:33:34.937Z | null
+2024-10-23T12:27:28.948Z | null
+2024-10-23T12:15:03.360Z | null
+;
+
+# same comment as above (fieldIsUnmappedButSourceIsDisabledSingleIndex)
+fieldIsUnmappedButExcludedFromSourceSingleIndex-Ignore
+required_capability: source_field_mapping
+required_capability: unmapped_fields
+required_capability: optional_fields
+
+SET unmapped_fields="load"\;
+FROM partial_mapping_excluded_source_sample_data
+| SORT @timestamp DESC
```

Comment:
If you want `message` you can have it with `KEEP message`, right? Or you could do something to do and it'd "appear", right?